src/15_classes.py

```diff
@@ -2,28 +2,46 @@
 # constructor
 
 # YOUR CODE HERE
+class LatLon:
+    def __init__(self, lat, lon):
+        self.lat = lat
+        self.lon = lon 
+
+ 
+
 
 # Make a class Waypoint that can be passed parameters `name`, `lat`, and `lon` to the
 # constructor. It should inherit from LatLon. Look up the `super` method.
 
 # YOUR CODE HERE
-
+class Waypoint(LatLon):
+    def __init__(self, name, lat, lon):
+        super().__init__(lat, lon)
+        self.name = name
+    def __str__(self):
+        return (f"Name: {self.name} lat: {str(self.lat)} lon: {str(self.lon)}") 
```

Comment:
There is a thread started by Trevor on various ways to use __str__ otherwise well done